static-files/oci_static_data.yml

```diff
@@ -1,10 +1,10 @@
 ---
 generators:
   - OCIComputeGenerator:
-      start_date: 2022-08-01
-      end_date: 2022-09-12
-      unit_price: 0.341
-      consumed_quantity: 88583
+      start_date: 2023-03-01
+      end_date: 2023-03-20
```

Comment:
Do we need to change the dates here?

Reply:
we don't need to, but I happened to be cleaning up